### PR TITLE
[new] Aiven Client Install instructions for MST

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Installing Aiven Client is important for tasks such as adding plugins and others. Hence, created this new instrucstions.

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:
Note: the child page to create a fork is in this [PR](https://github.com/timescale/docs/pull/1503/)

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: mst/aiven-client.md/aiven-client-install.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+Client you need to first create an authentication token and then configure it to
+connect to your Managed Service for TimescaleDB using the command line.
+
+## Create an authentication token in Managed Service for TimescaleDB
+
+To connect to Managed Service for TimecaleDB using Aiven Client you need to
+create an authentication token.
+
+<procedure>
+
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:19]:
```suggestion

### Creating an authentication token in Managed Service for TimescaleDB

```

File: mst/aiven-client.md/aiven-client-install.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+1.  Login to your [Managed Service for TimescaleDB portal][mst-login].
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:20]:
```suggestion
1.  Log in to your [Managed Service for TimescaleDB portal][mst-login].
```

File: mst/aiven-client.md/aiven-client-install.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+1.  Click `User Information` icon in the top left corner.
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:21]:
```suggestion
1.  Click `User Information` in the top right corner.
```

File: mst/aiven-client.md/aiven-client-install.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+1.  In the `User Profile` page, select the `Authentication`tab.
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:22]:
```suggestion
1.  In the `User Profile` page, navigate to the `Authentication`tab.
```

File: mst/aiven-client.md/aiven-client-install.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+1.  Click the `Generate Token` button.
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:23]:
```suggestion
1.  Click `Generate Token`.
```

File: mst/aiven-client.md/aiven-client-install.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+1.  Copy the generated authentication token and save it.
+
+</procedure>
+
+## Install and configure Aiven Client and connect to Managed Service for TimescaleDB
+
+The `avn` utility is a Python package, so you can install using pip on a Linux,
+MacOS or a Windows system which have Python installed, using `pip install
+aiven-client` command. For more information about installaing Aiven Client, see
+the [getting started][aiven-github] documentation.
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:35]:
```suggestion
Aiven Client is provided as a Python package, so you can install it on a Linux,
MacOS, or Windows system using `pip`, if you have already installed Python. 

Use this command: 

```bash
pip install aiven-client
```

For more information about installing the Aiven Client, see the 
[Aiven][aiven-github] documentation.
```

The GH webUI has broken my suggestion, but I'm sure you get the idea.

File: mst/aiven-client.md/aiven-client-install.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+### Configuring Aiven Client to connect to Managed Service for TimescaleDB
+
+To access Managed Service for TimescaleDB you need an authentication token.
+Aiven Client uses this authentication token to access your services on Managed
+Service for TimescaleDB.
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:41]:
```suggestion
To access Managed Service for TimescaleDB with the Aiven Client, you need an 
authentication token. Aiven Client uses this authentication token to access your 
services on Managed Service for TimescaleDB.
```

File: mst/aiven-client.md/aiven-client-install.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+<procedure>
+
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:44]:
```suggestion

### Configuring Aiven Client to connect to Managed Service for TimescaleDB

```

File: mst/aiven-client.md/aiven-client-install.md [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+</procedure>
+
+## Install and configure Aiven Client and connect to Managed Service for TimescaleDB
+
+The `avn` utility is a Python package, so you can install using pip on a Linux,
+MacOS or a Windows system which have Python installed, using `pip install
+aiven-client` command. For more information about installaing Aiven Client, see
+the [getting started][aiven-github] documentation.
+
+### Configuring Aiven Client to connect to Managed Service for TimescaleDB
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:37]:
```suggestion
## Configure Aiven Client to connect to Managed Service for TimescaleDB
```

File: mst/aiven-client.md/aiven-client-install.md [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+1.  Click `User Information` icon in the top left corner.
+1.  In the `User Profile` page, select the `Authentication`tab.
+1.  Click the `Generate Token` button.
+1.  In the `Generate access token` dialog, type a descriptive name for the
+    token and leave the rest of the fields blank.
+1.  Copy the generated authentication token and save it.
+
+</procedure>
+
+## Install and configure Aiven Client and connect to Managed Service for TimescaleDB
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:30]:
```suggestion
## Install the Aiven Client
```

File: mst/aiven-client.md/aiven-client-install.md [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+
+<procedure>
+
+1.  Change to the install directory that contains the configuration files:
+
+    ```bash
+    cd ~/.config/aiven/
+    ```
+
+1.  Open the `aiven-credentials.json` using any editor and update the following
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:51]:
```suggestion
1.  Open the `aiven-credentials.json` using any editor and update these
```

File: mst/aiven-client.md/aiven-client-install.md [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+    lines with your Managed Service for TimescaleDB `User email` and
+    `authentication token` that you generated :
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:53]:
```suggestion
    lines with your Managed Service for TimescaleDB `User email`, and the
    `authentication token` that you generated :
```

File: mst/aiven-client.md/aiven-client-install.md [hunk 13]
```diff
@@ -0,0 +1,85 @@
...
+1.  Save the `aiven-credentials.json` file.
+
+1.  To verify that you can access your services on Managed Service for
+    TimescaleDB, type:
+
+    ```bash
+    avn project list
+    ```
+
+    A list of all your projects appear.
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:71]:
```suggestion
    This command shows a list of all your projects:
```

File: mst/aiven-client.md/aiven-client-install.md [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+
+    ```bash
+    
+    PROJECT_NAME       DEFAULT_CLOUD            CREDIT_CARD
+    =================  =======================  ===========
+    internal-90d0      timescale-aws-us-east-1  N/A
+    timescale-6d6d     timescale-aws-us-east-1  N/A
+    docs-test-project  timescale-aws-us-east-1  N/A
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:79]:
```suggestion
    project-xxxx      timescale-aws-us-east-1  N/A
    project-yyyy     timescale-aws-us-east-1  N/A
    project-zzzz.    timescale-aws-us-east-1  N/A
```

Comment (Loquacity) [reply mst/aiven-client.md/aiven-client-install.md:79]: Or, alternatively, we just remove the example output for this command. I'm not sure I'm happy having it show "N/A" for the credit card field.

File: mst/aiven-client.md/index.md [hunk 15]
```diff
@@ -0,0 +1,19 @@
+---
+title: Aiven Client for Managed Service for TimescaleDB
+excerpt: A Command Line tool for fully managed TimescaleDB service on AWS, Azure, or GCP.
+product: mst
+---
+
+# Aiven Client for Managed Service for TimescaleDB
+
+[Aiven](https://aiven.io/) Client is a is a tool that you can install to manage
```
Comment (Loquacity) [inline mst/aiven-client.md/index.md:9]: Make this link reference style

Comment (Loquacity) [reply mst/aiven-client.md/index.md:9]:
```suggestion
You can use [Aiven Client][aiven-client] to manage
```

File: mst/aiven-client.md/index.md [hunk 16]
```diff
@@ -0,0 +1,19 @@
...
+your services on Managed Service for TimescaleDB.
+
+aiven-client (avn) is the command-line client for Managed Service for TimescaleDB.
```
Comment (Loquacity) [inline mst/aiven-client.md/index.md:12]:
```suggestion
```

This is just a repeat of the previous para.

File: mst/aiven-client.md/index.md [hunk 17]
```diff
@@ -0,0 +1,19 @@
...
+
```
Comment (Loquacity) [inline mst/aiven-client.md/index.md:13]:
```suggestion
You can use the Aiven Client tool to:
```

File: mst/aiven-client.md/index.md [hunk 18]
```diff
@@ -0,0 +1,19 @@
...
+*   Install [aiven-client][aiven-install] and connect to Managed Service for TimescaleDB
```
Comment (Loquacity) [inline mst/aiven-client.md/index.md:14]:
```suggestion
*   Connect to Managed Service for TimescaleDB
```

File: mst/aiven-client.md/index.md [hunk 19]
```diff
@@ -0,0 +1,19 @@
...
+*   Create a service
+*   Create a fork
+*   Add authentication plugins to Grafana service
```
Comment (Loquacity) [inline mst/aiven-client.md/index.md:17]:
```suggestion
*   Add authentication plugins to your attached Grafana service
```

File: mst/page-index/page-index.js [hunk 20]
```diff
@@ -11,6 +11,10 @@ module.exports = [
         title: "About Managed Service for TimescaleDB",
         href: "about-mst",
       },
+      {
+        title: "Command Line tool for Managed Service for TimescaleDB",
```
Comment (Loquacity) [inline mst/page-index/page-index.js:15]:
```suggestion
        title: "Command line access",
```

File: mst/aiven-client.md/aiven-client-install.md (hunk 14 above)
Comment (Rajakavitha1) [reply mst/aiven-client.md/aiven-client-install.md:86]: should we make that credit card 'xxxx-xxxx-xxxx-xxxx'?

File: mst/aiven-client.md/aiven-client-install.md [hunk 21]
```diff
@@ -0,0 +1,92 @@
...
+## Install the Aiven Client
+
+Aiven Client is provided as a Python package, so you can install it on a Linux,
+MacOS, or Windows system using `pip`, if you have already installed Python. 
+
+Use this command: 
+
+```bash
+pip install aiven-client
+
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:41]: Need to add in the rest here ...

File: mst/aiven-client.md/aiven-client-install.md (hunk 14 above)
Comment (Loquacity) [reply mst/aiven-client.md/aiven-client-install.md:86]: Yeah, that might work 👍

File: mst/aiven-client.md/index.md [hunk 22]
```diff
@@ -0,0 +1,18 @@
...
+You can use [Aiven Client][aiven-client] to manage
+your services on Managed Service for TimescaleDB.
+
+You can use the Aiven Client tool to:
+*   Connect to Managed Service for TimescaleDB
+*   Create a service
+*   Create a fork
+*   Add authentication plugins to your attached Grafana service
+
+[aiven-install]: /mst/:currentVersion:/aiven-client-install/
```
Comment (Loquacity) [inline mst/aiven-client.md/index.md:18]:
```suggestion
[aiven-install]: /mst/:currentVersion:/aiven-client-install/
[aiven-client]: FIXME
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: mst/aiven-client.md/aiven-client-install.md (hunk 14 above)
Comment (Rajakavitha1) [reply mst/aiven-client.md/aiven-client-install.md:86]: done!!!!

File: mst/aiven-client.md/aiven-client-install.md (hunk 21 above)
Comment (Loquacity) [reply mst/aiven-client.md/aiven-client-install.md:42]: Wasn't there another line after the code block?

File: mst/aiven-client.md/aiven-client-install.md [hunk 23]
```diff
@@ -0,0 +1,93 @@
...
+    ```bash
+    avn project list
+    ```
+
+    This command shows a list of all your projects:
+
+    ```bash
+    
+    PROJECT_NAME       DEFAULT_CLOUD            CREDIT_CARD
+    =================  =======================  ===========
```
Comment (Loquacity) [inline mst/aiven-client.md/aiven-client-install.md:84]: Might need to adjust these to line up as well.

File: mst/aiven-client.md/aiven-client-install.md (hunk 21 above)
Comment (Rajakavitha1) [reply mst/aiven-client.md/aiven-client-install.md:42]: Not it is a just one line command

File: mst/aiven-client.md/aiven-client-install.md (hunk 23 above)
Comment (Rajakavitha1) [reply mst/aiven-client.md/aiven-client-install.md]: adjusted

File: mst/aiven-client.md/aiven-client-install.md (hunk 21 above)
Comment (Loquacity) [reply mst/aiven-client.md/aiven-client-install.md:42]: Nope, found it:

```suggestion
For more information about installing the Aiven Client, see the
[Aiven][aiven-github] documentation.
```

**Review by Loquacity — APPROVED** [review]

File: mst/page-index/page-index.js [hunk 24]
```diff
@@ -11,6 +11,10 @@ module.exports = [
         title: "About Managed Service for TimescaleDB",
         href: "about-mst",
       },
+      {
```
Comment (charislam) [inline mst/page-index/page-index.js:14]: The second page is missing from the page-index :)

Comment (Rajakavitha1) [reply mst/page-index/page-index.js:14]: added the child pages

File: mst/page-index/page-index.js [hunk 25]
```diff
@@ -11,6 +11,19 @@ module.exports = [
...
+        children: [
+        {
+        title: "Aiven Client for Managed Service for TimescaleDB",
+        href: "aiven-client-install",
+        },
+        {
+        title: "Create a fork using Aiven Client",
+        href: "create-fork",
+        },
+       ],
```
Comment (charislam) [inline mst/page-index/page-index.js:26]: I think there is a missing curly brace here, can you indent these like the other entries? It makes it really hard to check bracket closing if they aren't indented 🙏🏼

**Review by charislam — CHANGES_REQUESTED** [review]

File: mst/page-index/page-index.js (hunk 25 above)
Comment (Loquacity) [reply mst/page-index/page-index.js:26]: preview is returning 403 for me ...

Comment (Rajakavitha1) [reply mst/page-index/page-index.js:26]: Screenshot 2022-08-29 at 8 00 07 AM
Same here!!!

Comment (Loquacity) [reply mst/page-index/page-index.js:26]: I'm looking at it now, Rajie.

Comment (Loquacity) [reply mst/page-index/page-index.js]: I have it working as a single page:

image

We can either merge like this (if you want), or I can keep trying to get it into a nested page after lunch 🥪

Comment (Rajakavitha1) [reply mst/page-index/page-index.js]: Let's merge it @Loquacity  considering that we have three nested pages, install cli, fork service, and auth plugin pages. We can may be have a new PR for nested pages after the new pages are merged. Thank you so much @Loquacity .

Comment (Loquacity) [reply mst/page-index/page-index.js]: np. we can work it out with the next PR 👍

**Review by charislam — APPROVED** [review]

**Comment by Loquacity** [comment]
> I think we need an actual engineer to test this one. We have issues with the CLI so I'd prefer we double check it

@Rajakavitha1 has gotten this working, and I'm happy to run a test as well, if you think it needs more eyes. What kind of issues have you had with it?

**Comment by Loquacity** [comment]
I'm going to go ahead and merge this, because we have failing links without it.

**Pull request merged**